docs/io/fits/index.rst

```diff
@@ -47,6 +47,10 @@ Once the `astropy.io.fits` package is loaded using the standard convention
 
     >>> hdul = fits.open(fits_image_filename)
 
+Note : The :func:`get_testdata_filepath` function is only used by tests to get their data.
```

Comment:
sphinx cannot interpret function names on their own, full namespace path is needed. However it's possible to show only the function name in the docs by starting the name with a `~`. E.g.:
`~astropy.io.fits.util.get_testdata_filepath`